### ISSUE TITLE
Keyboard accessibility issue with `stache-action-buttons`

### ISSUE THREAD

**Issue by Blackbaud-ScottFreeman** [blackbaud/skyux-lib-stache#54]
### Expected behavior
A keyboard user should be able to filter action buttons using the search and tab to the filtered selection.

### Actual behavior
Tabbing erases the search text and removes the filtering.

### Steps to reproduce
Go to the skyux components page.
https://developer.blackbaud.com/skyux/components

Try searching for a component and tabbing to the action button to see the search text disappear.

**Comment by Blackbaud-LindseyRix** [comment]
We are planning to switch to the SKY action buttons soon, just waiting on one feature from them. So we won't be addressing this work in Stache. Thanks for providing the feedback!
